superduperdb/ext/torch/model.py

```diff
@@ -404,6 +408,9 @@ def parameters(self):
     def state_dict(self):
         return self.object.state_dict()
 
+    def to_method(self, device):
+        self.object.artifact.to(device)
```

Comment:
This assumes that every framework uses `.to`. (Might not be the case.)

Reply:
Addressed :)